srcpkgs/cargo-tauri/template

```diff
@@ -0,0 +1,19 @@
+# Template file for 'cargo-tauri'
+pkgname=cargo-tauri
+version=1.4.0
+revision=1
+build_wrksrc="tooling/cli"
+build_style=cargo
+hostmakedepends="pkg-config"
+makedepends="bzip2-devel libzstd-devel openssl-devel"
+short_desc="Command line interface for building Tauri apps"
+maintainer="Marcin Puc <tranzystorek.io@protonmail.com>"
+license="Apache-2.0, MIT"
+homepage="https://tauri.app"
+changelog="https://raw.githubusercontent.com/tauri-apps/tauri/dev/tooling/cli/CHANGELOG.md"
+distfiles="https://github.com/tauri-apps/tauri/archive/refs/tags/tauri-cli-v${version}.tar.gz"
+checksum=4ac0985d750f7a4c6849ecd4bcdd26306b5dcf99c84f71762a038b290afbf781
+
+post_install() {
+	vlicense LICENSE_MIT
```

Comment:
You can generate completions apparently: https://tauri.app/v1/api/cli/#completions

Reply:
Yeah, but for cargo plugins they are effectively useless...

Reply:
Fair enough.